docs/framework/wpf/advanced/opentype-font-features.md

```diff
@@ -191,22 +188,22 @@ Text using OpenType contextual swashes
 ## Alternates  
  Alternates are glyphs that can be substituted for a standard glyph. [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] fonts, such as the Pericles font used in the following examples, can contain alternate glyphs that you can use to create different appearances for text. The following text displays standard glyphs for the Pericles font.  
   
- ![Text using OpenType standard glyphs](../../../../docs/framework/wpf/advanced/media/opentypefont01.gif "opentypefont01")  
-Text using OpenType standard glyphs  
+ ![Text using OpenType standard glyphs](./media/opentype-standard-glyphs.gif "Text using OpenType standard glyphs")  
+  
   
  The Pericles [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] font contains additional glyphs that provide stylistic alternates to the standard set of glyphs. The following text displays stylistic alternate glyphs.  
   
- ![Text using OpenType stylistic alternate glyphs](../../../../docs/framework/wpf/advanced/media/opentypefont02.gif "opentypefont02")  
```

Comment:
it seems that this one and another image were also being used by another topic. So let's create the copy as we discussed associated with that particular article.

Reply:
Images copied to directory for the additional article.